### DISCUSSION TITLE
"Missing plug-in" Unable to record audio

### DISCUSSION THREAD

**Discussion by rlhugo** [SafeExamBrowser/seb-win-refactoring#477] [Q&A]
We are running SEB in Moodle, with the Poodll plugin. (HTML5 audio recording questions). Students are getting a 'missing plug-in' message for it. Is there a way to allow tools like Poodll in the config file? I can't seem to see where the setting would be (as it is not a local process). Thanks!

**Comment by dbuechel** [comment]
Is Poodll a Moodle or a browser plugin? If it is the latter, then that's correct, SEB does (currently) not support browser plugins. Apart from that, with which SEB version are you experiencing the issue?
